Documentation/k8s-monitoring.md

```diff
@@ -0,0 +1,50 @@
+# Monitoring
+Each Rook cluster has some built in metrics collectors/exporters for monitoring with [Prometheus](https://prometheus.io/).  To enable monitoring of Rook in your Kubernetes cluster, you can follow the steps below.
+Note that these steps work best with a local Kubernetes cluster running in `Vagrant`.
+
+## Prometheus Operator
+First the Prometheus operator needs to be started in the cluster so it can watch for our requests to start monitoring Rook and respond by deploying the correct Prometheus pods and configuration.
+A full explanation can be found in the [Prometheus operator repository on github](https://github.com/coreos/prometheus-operator), but the quick instructions can be found here:
+```
+pushd /tmp
+git clone https://github.com/coreos/prometheus-operator.git
```

Comment:
this may be an easy way: `wget -qO- https://raw.githubusercontent.com/coreos/prometheus-operator/master/deployment.yaml | kubectl create -f -`